integration-tests/test-app/sub-mirror-input.js

```diff
@@ -8,14 +8,17 @@ const html = registerHtml({
  * component to test url parameters
  */
 module.exports = () => {
-	const mirrorable = useGlobalStore('mirrorable-input', { value: '' })
```

Comment:
I don't think we actually needed this `, { value: '' }`, it was documented as an error #156 , but this works without it

Reply:
In reality, I just need to create a minimal example of this...